src/main/views/privacy-policy.njk

```diff
@@ -0,0 +1,125 @@
+{% extends "layout.njk" %}
+{% from "support.njk" import supportWidget %}
+{% set heading = t('Privacy policy') %}
+
+{% set title = t('Money claim') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <p>{{ t('When you use the money claims service we will ask you for some personal data.' +
+        'This privacy policy explains why we collect your personal data, what we do with it and your rights.') }}
+      </p>
+      <p>
+        {{ t('The money claim service is managed by Her Majesty’s Courts and Tribunals Service and '
+        + 'will be referred to as we from now on.') }}
+      </p>
+
+      <p> {{ t('More information about how to use the service is in the '
+        +'<a href="{{  termsAndConditions }}">terms and conditions</a>.',
```

Comment:
Please use `AppPaths` here as well. Since whitespace after `{{` is enough.

Reply:
👍